crates/provider/src/shared/scripts/helper-binaries-downloader.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/ash
+
+log() {
+    echo "$(date +"%F %T") $1"
+}
+
+# used to handle the distinction where /cfg is used for k8s and /helpers for docker/podman
+# to share a volume across nodes containing helper binaries and independent from /cfg
+# where some node files are stored
+OUTDIR=$([ -d /helpers ] && echo "/helpers" || echo "/cfg")
```

Comment:
This should be _noticed_ when we run scripts since we relay on the default `/cfg/[curl/coreutils]`